Resources/Prototypes/Body/Organs/arachnid.yml

```diff
@@ -52,120 +27,44 @@
     updateInterval: 1.5
-    removeEmpty: true
-    solutionOnBody: false
-    solution: "Lung"
-    metabolizerTypes: [ Human ]
-    groups:
-    - id: Gas
-      rateModifier: 100.0
-  - type: SolutionContainerManager
-    solutions:
-      organ:
-        reagents:
-        - ReagentId: Nutriment
-          Quantity: 10
-      Lung:
-        maxVol: 100.0
-        canReact: false
-      food:
-        maxVol: 5
-        reagents:
-        - ReagentId: UncookedAnimalProteins
-          Quantity: 5
 
 - type: entity
+  parent: [BaseOrganHeart, BaseArachnidOrgan]
   id: OrganArachnidHeart
-  parent: BaseArachnidOrgan
-  name: heart
-  description: "A disgustingly persistent little biological pump made for spiders."
+  description: A disgustingly persistent little biological pump made for spiders.
   components:
-  - type: Sprite
-    state: heart-on
-  - type: Item
-    size: Small
-    heldPrefix: heart
   - type: Metabolizer
     updateInterval: 1.5
-    maxReagents: 2
     metabolizerTypes: [Arachnid]
-    groups:
-    - id: Medicine
-    - id: Poison
-    - id: Narcotic
 
 - type: entity
   id: OrganArachnidLiver
-  parent: BaseHumanOrgan
-  name: liver
-  description: "Pairing suggestion: chianti and fava beans."
+  parent: [BaseHumanOrgan, OrganAnimalLiver, BaseArachnidOrgan] #TODO: sprite arachnid liver and change parent from BaseHumanOrgan to BaseArachnidOrgan
+  description: 'Pairing suggestion: chianti and fava beans.'
   categories: [ HideSpawnMenu ]
   components:
-  - type: Item
-    size: Small
-    heldPrefix: liver
-  - type: Sprite
-    state: liver
-  - type: Metabolizer # The liver metabolizes certain chemicals only, like alcohol.
+  - type: Metabolizer
     updateInterval: 1.5
-    maxReagents: 1
-    metabolizerTypes: [Animal]
-    groups:
-    - id: Alcohol
-      rateModifier: 0.1 # removes alcohol very slowly along with the stomach removing it as a drink
 
 - type: entity
+  parent: [ BaseHumanOrgan, OrganAnimalKidneys, BaseArachnidOrgan] #TODO: sprite arachnid kidneys and change parent from BaseHumanOrgan to BaseArachnidOrgan
   id: OrganArachnidKidneys
-  parent: BaseHumanOrgan
-  name: kidneys
-  description: "Filters toxins from the bloodstream."
   categories: [ HideSpawnMenu ]
   components:
-  - type: Sprite
-    layers:
-      - state: kidney-l
-      - state: kidney-r
-  # The kidneys just remove anything that doesn't currently have any metabolisms, as a stopgap.
-  - type: Item
-    size: Small
-    heldPrefix: kidneys
   - type: Metabolizer
     updateInterval: 1.5
-    maxReagents: 5
-    metabolizerTypes: [Animal]
-    removeEmpty: true
 
 - type: entity
+  parent: [BaseOrganEyes, BaseArachnidOrgan]
   id: OrganArachnidEyes
-  parent: BaseArachnidOrgan
-  name: eyes
-  description: "Two was already too many."
-  components:
-  - type: Sprite
-    layers:
-      - state: eyeball-l
-      - state: eyeball-r
-  - type: Item
-    size: Small
-    heldPrefix: eyeballs
+  description: Two was already too many.
```

Comment:
Check the arachnid inheritances, something is broken and they get the human sprites. Is it maybe not predictable/deterministic which sprite will be used if both parents specify one? 
![image](https://github.com/user-attachments/assets/ef695ff5-aae6-4867-9227-26492d175a77)

On my branch I specified the sprite on Base((Species))Organ and the state on the BaseOrgan((Organ)) so there was never a possible conflict, but that does require setting up Base((Species))Organ for all the species even if we just use the human sprites

Reply:
you have to flip the parents

Reply:
(The other way to fix this and also avoid it ever happening again on organs would be if the parents were always strictly: sprite from one parent, state from the other, as discussed below)